### DISCUSSION TITLE
No guess mode?

### DISCUSSION THREAD

**Discussion by MIC132** [Mikubill/sd-webui-controlnet#1690] [Q&A] [answered]
So I recently started using the controlnet plugin for webui, but I can't seem to find the "guess mode" which is mentioned in the readme. The screenshots seem to be from an older version and the checkbox is nowhere to be found in the current one. Was it removed, or am I just blind?

**Comment by huchenlei** [comment, marked as answer]
Set control mode to 'controlnet is more important' is equivalent to 'guess mode'
https://github.com/Mikubill/sd-webui-controlnet/blob/6dc9f34b98033764bc2fe6a87aaa26d98e9e37b5/README.md?plain=1#L68

**Reply by MIC132** [reply]
Oh, I completely missed that. Thanks!
